astroquery/desi/tests/test_module_remote.py

```diff
@@ -0,0 +1,65 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import time
+
+import pytest
+from astropy.io.fits import HDUList
+from astroquery.exceptions import NoResultsWarning
+
+
+@pytest.mark.remote_data
+class TestLegacySurveyClass:
+
+    def test_query_region(self):
+        t0 = time.time()
+        print("Beginning test")
+        import astroquery.desi
+        from astropy.coordinates import SkyCoord
+        from astropy.coordinates import Angle
+        from astropy.table import Table
+
+        ra = Angle('11h04m27s', unit='hourangle').degree
+        dec = Angle('+38d12m32s', unit='hourangle').degree
+        coordinates = SkyCoord(ra, dec, unit='degree')
+
+        radius = Angle(5, unit='arcmin')
+
+        query1 = astroquery.desi.DESILegacySurvey.query_region(coordinates, radius=radius, data_release=9)
+        print("Test completion: ", time.time() - t0)
+        print(query1['ra', 'dec', 'type'])
```

Comment:
Remove print statements in the testing suite (only mentioning it here, but remove them throughout).